drivers/serial/uart_nrfx_uarte.c

```diff
@@ -1954,6 +1948,11 @@ static int uarte_nrfx_pm_control(const struct device *dev,
 		    (UARTE_PROP(idx, pin_prop)),			       \
 		    (NRF_UARTE_PSEL_DISCONNECTED))
 
+#define UARTE_PULL(idx, pull_up_prop) \
+	COND_CODE_1(UARTE_HAS_PROP(idx, pull_up_prop), \
```

Comment:
In general DT_PROP should be used instead of DT_HAS_PROP in the case of boolean properties. It will expand to 0 or 1 depending on if the property is present.

Reply:
Changed to `UARTE_PROP`.